src/components/header/nav.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Link } from "@mui/material";
+import { Auth } from "../../atoms/auth";
+import { NavButton } from "./navbutton";
+import strings from "../../localization/strings";
```

Comment:
```suggestion
import strings from "../../localization/strings";

```

Reply:
Adding a line break between imports and first line of code

Reply:
This would be really nice if Rome were to be configured to do this for me